app/Http/Controllers/QuizVerificationController.php

```diff
@@ -0,0 +1,23 @@
+<?php
+
+namespace App\Http\Controllers;
+
+use Illuminate\Http\Request;
+
+class QuizVerificationController extends Controller
+{
+    public function showVerificationForm(Quiz $quiz)
+    {
+        return view('quizzes.verify')->withQuiz($quiz);
```

Comment:
I'd prefer passing a second argument to view helper using `compact`, instead of chaining `withQuiz` method.

Reply:
But I prefer those "magicfunctions", do you want me to use `compact` instead?

Reply:
Its about consistency, I have never used magic function to pass the data to view, not even when there is only one single value. So, its better if you maintain that. Or, change every other reference to use .agic functions instead, but dont mix them.